Modules/User/Http/Requests/roleRequest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Modules\User\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class RoleRequest extends FormRequest
+{
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required|string|unique:roles,name',
+            'guard_name' => 'required|string:roles,guard_name',
+            'description' => 'required|string:roles,description',
```

Comment:
What this means `string:roles,guard_name`? Its not even a rule.
```suggestion
            'guard_name' => 'required|string',
            'description' => 'required|string',
```